pom.xml

```diff
@@ -64,7 +64,8 @@
     <properties>
         <revision>2.21</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.164.3</jenkins.version>
+        <jenkins.version>2.241-rc30039.aafbea2368b4</jenkins.version>
+ <!-- https://github.com/jenkinsci/jenkins/pull/4784 -->
```

Comment:
```suggestion
        <jenkins.version>2.241-rc30039.aafbea2368b4</jenkins.version> <!-- https://github.com/jenkinsci/jenkins/pull/4784 -->
```

Reply:
Was intended to be on the same line. Will adjust this when I update this PR to use a newer incremental.

Reply:
Now done